### PR TITLE
Add lifetime to dataset from ConfigFile

### PR DESCRIPTION
**Pull request by jhonatanamado**
Add lifetime for dataset (RucioContainerRules) that will be subscribed to only disk sites via our ConfigFile

### CODE REVIEW

File: src/python/T0/RunConfig/RunConfigAPI.py [hunk 1]
```diff
@@ -689,11 +693,13 @@ def configureRunStream(tier0Config, run, stream, specDirectory, dqmUploadProxy):
             factory = RepackWorkloadFactory()
             wmSpec = factory.factoryWorkloadConstruction(workflowName, specArguments)
             for subscription in subscriptions:
+                logging.info("subscrition_bulk %s", subscription)
```
Comment (germanfgv) [inline src/python/T0/RunConfig/RunConfigAPI.py:696]: Typo

File: src/python/T0/RunConfig/RunConfigAPI.py [hunk 2]
```diff
@@ -689,11 +693,13 @@ def configureRunStream(tier0Config, run, stream, specDirectory, dqmUploadProxy):
...
                 wmSpec.setSubscriptionInformation(**subscription)
         elif streamConfig.ProcessingStyle == "Express":
             factory = ExpressWorkloadFactory()
             wmSpec = factory.factoryWorkloadConstruction(workflowName, specArguments)
             for subscription in subscriptions:
+                logging.info("subscrition_express %s", subscription)
```
Comment (germanfgv) [inline src/python/T0/RunConfig/RunConfigAPI.py:702]: Also typo. I'd also review the need for this log entry, and if you consider it to be necessary, then please give it a clearer format so anyone can understand what information is being given.

**Review by germanfgv — COMMENTED** [review]
Looks good, just a couple of comments about logging

File: src/python/T0/RunConfig/RunConfigAPI.py (hunk 1 above)
Comment (jhonatanamado) [reply src/python/T0/RunConfig/RunConfigAPI.py]: This was only for debugging purposes. I updated this PR

File: src/python/T0/RunConfig/RunConfigAPI.py (hunk 2 above)
Comment (jhonatanamado) [reply src/python/T0/RunConfig/RunConfigAPI.py]: Same as above

**Review by germanfgv — APPROVED** [review]
This is ready to be merged

**Comment by germanfgv** [comment]
@jeyserma Can you comment on the default lifetime that we should give to our subscriptions?

**Comment by drkovalskyi** [comment]
Let's set 3 months as default till we have a better number.

**Pull request merged**